Temporarily disable the calendarhandler test.
This test has been fixed upstream, so this should be able to be
 removed in the next release.

diff -Naur a/autotests/CMakeLists.txt b/autotests/CMakeLists.txt
--- a/autotests/CMakeLists.txt	2022-04-18 11:34:57.356111303 -0700
+++ b/autotests/CMakeLists.txt	2022-04-18 11:33:56.175938220 -0700
@@ -35,7 +35,7 @@
 ecm_add_test(postprocessortest.cpp LINK_LIBRARIES Qt::Test KPim::Itinerary)
 ecm_add_test(extractorvalidatortest.cpp LINK_LIBRARIES Qt::Test KPim::Itinerary)
 if (TARGET KF5::CalendarCore)
-    ecm_add_test(calendarhandlertest.cpp LINK_LIBRARIES Qt::Test KPim::Itinerary)
+   #ecm_add_test(calendarhandlertest.cpp LINK_LIBRARIES Qt::Test KPim::Itinerary)
     ecm_add_test(extractortest.cpp LINK_LIBRARIES Qt::Test KPim::Itinerary KPim::PkPass)
 endif()
 ecm_add_test(documentutiltest.cpp LINK_LIBRARIES Qt::Test KPim::Itinerary)
